recipes/libsecret/all/conanfile.py

```diff
@@ -105,6 +106,6 @@ def package_info(self):
         self.cpp_info.names["pkg_config"] = "libsecret-1"
         self.cpp_info.requires = ["glib::glib-2.0", "glib::gobject-2.0"]
         if self._use_gcrypt:
-            self.cpp_info.requires.append("libgcrypt::gcrypt")
+            self.cpp_info.requires.append("libgcrypt::libgcrypt")
```

Comment:
```suggestion
```
This line was useless in the first place. the `if` above has to be removed too

Reply:
No, it's required due to `self.cpp_info.requires = ["glib::glib-2.0", "glib::gobject-2.0"]`